src/styles/utils/_loops.scss

```diff
@@ -0,0 +1,24 @@
+// Counter for each cell of the day
+@for $i from 1 through 31 {
```

Comment:
you can make class names for days with loop and array, try to implement it

Reply:
Thanks for your comments, they  teach me a lot.